modules/upgrade/pages/migrate-4x-4x.adoc

```diff
@@ -45,22 +46,28 @@ If there are older updates available, {yast} will notify you and ask to install
 You must install all package updates before performing the migration.
 For more information, see xref:server-update.adoc[].
 +
-////
-TODO:
-2019-10-01, ke:
-With the advent of SLE 15 SP2 or SUMA 4.1 we probably must enhance this;
-For more info, see the maintenance/3.2 branch!
-Enhance the zypper section below accordingly
-////
 {yast} will show the possible migration targets with detailed summaries.
 // I've removed the graphical option, because we tell people to use a text console. LKB 2019-08-21
 . Select the appropriate target, and follow the prompts to complete the migration.
 . Reboot the server.
+. When rebooted the spacewalk services are not running until you have migrated
+the PostgreSQL database to version{nbsp}12.
+. Log in on the text console as root, and run the database migration script:
++
+----
+/usr/lib/susemanager/bin/pg-migrate-10-to-12.sh
+----
```

Comment:
Maybe add a reference to the DB migration chapter?

Reply:
ok